sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/properties/profile/AzureEnvironmentProperties.java

```diff
@@ -25,13 +25,6 @@ public class AzureEnvironmentProperties implements AzureProfileOptionsProvider.A
             setServiceBusDomainName("servicebus.chinacloudapi.cn");
         }};
 
-    /**
-     * Provides the environment instance for Azure Germany.
-     */
-    public static final AzureEnvironmentProperties AZURE_GERMANY = new AzureEnvironmentProperties(AzureEnvironment.AZURE_GERMANY) {{
-            setServiceBusDomainName("servicebus.cloudapi.de");
-        }};
-
```

Comment:
Has this been released in previous version? If so, we shouldn't delete this, instead we should mark it as deprecated.